docs/DeveloperGuide.md

```diff
@@ -257,27 +257,46 @@ _{Explain here how the data archiving feature will be implemented}_
+|----------|----------------------------------------------|-------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
+| `* * *`  | business owner                               | add orders easily                                                       | keep track of ongoing orders                                                      |
+| `* * *`  | business owner                               | delete entire orders                                                    |                                                                                   |
+| `* * *`  | busy business owner                          | list all orders                                                         | see all my ongoing orders in one place and keep track of them                     |
+| `* * *`  | business owner                               | add items to my inventory                                               | keep track of additions to my inventory                                           |
+| `* * *`  | business owner                               | delete items from my inventory                                          | have an updated list of inventory items                                           |
+| `* * *`  | business owner                               | list all items in my inventory                                          | see all items in my inventory in one place                                        |
+| `* * *`  | business owner                               | tag inventory items                                                     | keep an organised inventory of items                                              |
+| `* *  `  | busy business owner                          | find orders                                                             | locate an order without manually searching through the entire list                |
+| `* *  `  | business owner                               | find items in my inventory                                              | locate an item in my inventory without manually searching through the entire list |
+| `* * `   | business owner                               | edit order details                                                      | update changes to order                                                           |
+| `* *  `  | business owner                               | edit inventory item details                                             | update changes to inventory                                                       |
+| `* *  `  | business owner                               | see which order have been paid for                                      | press customers who have not paid for payment                                     |
```

Comment:
I believe it should be 'see which orders have been paid for'.

As for benefit, 'press customers for payment' should be sufficient